### DISCUSSION TITLE
Welcome to gui Discussions!

### DISCUSSION THREAD

**Discussion by hewanbing** [attpc-daq/gui#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by susu5767** [comment]
目前此项工作比较容易，SiTCP官网示例程序的数据已得到。（https://github.com/BeeBeansTechnologies/sitcpy）
**_如有需要，可以协助其他的工作，欢迎打扰_**。
![2022-07-22 19-04-40 的屏幕截图](https://user-images.githubusercontent.com/57052208/180427079-ca6123d5-d161-4904-ab5e-fc00103db75d.png)
